strings/tips.txt

```diff
@@ -98,12 +97,11 @@ As the Chief Engineer, you can spy on and even forge PDA communications with the
 As the Chief Engineer, your locker contains a jetpack upgrade that can be attached to hardsuits.
 As the Chief Engineer, the power flow control console in your office will show you APC infos and lets you control them remotely.
 As an Engineer, the supermatter shard is an extremely dangerous piece of equipment: touching it will disintegrate you. So will touching it with telepathy.
-As an Engineer, you can electrify grilles by placing wire "nodes" beneath them: the big seemingly unconnected bulges from a half completed wiring job.
-As an Engineer, return to Engineering once in a while to check on the engine and SMES cells. It's always a good idea to make sure containment isn't compromised.
+As an Engineer, you can electrify grilles by placing powered cables beneath them.
+As an Engineer, return to Engineering once in a while to check on the engine and SMES cells. It's always a good idea to make sure the supermatter isn't delaminating.
 As an Engineer, you can temporarily power the station solely with the solar arrays. They will provide just enough electricity to power the station for a while, however their output is just slightly below what is truely needed to consistently power the station.
 As an Engineer, you can cool the supermatter crystal by spraying it with a fire extinguisher. Only for the brave!
-As an Engineer, the Particle Accelerator that powers singularity engines is also effective at killing blobs!
```

Comment:
Why remove this one instead of changing the engine name? Aren't these the same things that power the supermatter? The lasers?

Reply:
The tip is talking about the PA, this thing 
![image](https://user-images.githubusercontent.com/58055496/112268425-30cf1d80-8c34-11eb-9326-94ae10437288.png)

Reply:
Ah, figured the lasers were called the same thing.